control/canonical.py

```diff
@@ -133,7 +133,7 @@ def observable_form(xsys):
     Wrz = obsv(zsys.A, zsys.C)
 
     # Transformation from one form to another
-    Tzx = inv(Wrz) * Wrx
+    Tzx = solve(Wrz, Wrx)  # matrix right division, Tzx = inv(Wrz) * Wrx
```

Comment:
Thanks, this looks right.  I see there are no tests for observable_form - would you mind adding one?

This would be left division in [Matlab's terminology](https://www.mathworks.com/help/matlab/ref/mldivide.html)

Reply:
you are right. See my update